.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = false
+indent_style = space
+indent_size = 4
+
+[{*.java, *.xml}]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespaces = true
```

Comment:
Can't this be global?

Reply:
Yeah...
So, the trailing_white_spaces should come in [*] ?